integration/hive/src/test/java/org/apache/carbondata/hive/HiveCarbonTest.java

```diff
@@ -211,6 +248,85 @@ public void testStructType() throws Exception {
     checkAnswer(carbonResult, hiveResult);
   }
 
+  private ArrayList<DimensionRawColumnChunk> getDimRawChunk(Integer blockindex)
+      throws IOException {
+    File rootPath = new File(HiveTestUtils.class.getResource("/").getPath() + "../../../..");
+    String storePath = rootPath.getAbsolutePath() + "/integration/hive/target/warehouse/warehouse/hive_carbon_table/";
+    CarbonFile[] dataFiles = FileFactory.getCarbonFile(storePath)
+        .listFiles(new CarbonFileFilter() {
+          @Override
+          public boolean accept(CarbonFile file) {
+            if (file.getName().endsWith(CarbonCommonConstants.FACT_FILE_EXT)) {
+              return true;
+            } else {
+              return false;
+            }
+          }
+        });
+    ArrayList<DimensionRawColumnChunk> dimensionRawColumnChunks = read(dataFiles[0].getAbsolutePath(),
+        blockindex);
+    return dimensionRawColumnChunks;
+  }
+
+  private ArrayList<DimensionRawColumnChunk> read(String filePath, Integer blockIndex) throws IOException {
```

Comment:
If this code to verify LocalDictionary is same as the other LocalDictionaryTest cases, please move to a common Util class and use at both places.

Reply:
Done.